### PR TITLE
Closes PBT-1129

### PR DESCRIPTION
**Pull request by vseufe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,11 @@ You can run your application in dev mode that enables live coding using:
 ./mvnw quarkus:dev
 ```
 
+## Build Docker image
+```
+docker build -f docker/Dockerfile.jvm -t t-learn/backend .
```
Comment (cbuettgentarent) [inline README.md:16]: should be `./mvnw clean package -Dquarkus.container-image.build=true`

File: README.md [hunk 2]
```diff
@@ -39,4 +44,4 @@ Or, if you don't have GraalVM installed, you can run the native executable build
 
 You can then execute your native executable with: `./target/t-learn-backend-1.0.0-SNAPSHOT-runner`
 
-If you want to learn more about building native executables, please consult https://quarkus.io/guides/building-native-image.
\ No newline at end of file
+If you want to learn more about building native executables, please consult https://quarkus.io/guides/building-native-image.
```
Comment (cbuettgentarent) [inline README.md:47]: what did change here?

File: docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+version: "3.7"
+services:
+  app-backend:
+    restart: always
+    image: t-learn/backend:latest
```
Comment (cbuettgentarent) [inline docker-compose.yml:5]: not sure the image name matches the one created by Quarkus (Quarkus also uses versions)

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    build: docker/Dockerfile.jvm
```
Comment (cbuettgentarent) [inline docker-compose.yml:6]: not needed, can't build the image anyway because the compiled application is not there on the server

**Review by cbuettgentarent — CHANGES_REQUESTED** [review]
Please leave the Dockerfiles where Quarkus expects them

File: README.md (hunk 2 above)
Comment (vseufe) [reply README.md:47]: It's githubs weird way of showing that there is now an empty line at the end of the file

File: README.md (hunk 1 above)
Comment (vseufe) [reply README.md]: For this I also removed the commit that moved the Docker directory, since Quarkus per default needs them to be in `/src/main`

**Review by cbuettgentarent — DISMISSED** [review]

**Review by cbuettgentarent — APPROVED** [review]

**Pull request merged**